### PR TITLE
Class4 Strings

### PR DESCRIPTION
**Pull request by MagaHovhannisyan**
Dear Smbat,
Sorry for being late.
Please find the PR related to Strings.

### CODE REVIEW

File: src/class4Strings/StringAPI.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        for (int i = 0; i < startWord.length; ++i) {
+            if (!startWord[i].isEmpty()) {
+                count++;
+            }
+        }
+        System.out.println("Number of words : " + (count));
+    }
+
+    //CHECK IF STRING IS PALINDROME
+    static void checkIfPalindrome(String getString) {
```
Comment (Smbat888) [inline src/class4Strings/StringAPI.java:67]: if (yourString.equals(new StringBuilder(yourString).reverse())) --> yes, it's polidrome!

:)

File: src/class4Strings/StringAPI.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            }
+        }
+        System.out.println("Palindrome");
+    }
+
+    //CHECK IF OPENING AND CLOSING PARANTHESIS ARE EQUAL
+    static void getParenthesis(String getString) {
+        for (int i = 0; i < getString.length(); ++i) {
+            //Error message when entering non parenthesis
+            if (getString.charAt(i) != '(' && getString.charAt(i) != ')') {
```
Comment (Smbat888) [inline src/class4Strings/StringAPI.java:81]: || instead of && in the if statement

File: src/class4Strings/StringAPI.java [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                System.out.println("You are allowed to enter only parenthesis");
+                return;
+            }
+        }
+        int open = 0;
+        int close = 0;
+        //Count opening and closing parenthesis
+        for (int i = 0; i < getString.length(); ++i) {
```
Comment (Smbat888) [inline src/class4Strings/StringAPI.java:89]: karair ham open-y ham close-y ham el invalid symbol-y mi hatik for-i mej aneir... else if-ov .. u ete hankarc for-ov fraluc handiper invalid symbol.. return aneir aseir senc chi lini))
du 2 for-ov es arel, chishty chishta, bayc avel gorcoxutyuna, hnenc el er kareli anel)

File: src/class4Strings/StringAPI.java [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        if (getText.endsWith(getContent)) {
+            System.out.println("true");
+        } else {
+            System.out.println("false");
+        }
+    }
+
+    //REMOVE DUPLICATE CHARACTERS
+    static void removeDuplicates(String word) {
+        String result = "";
```
Comment (Smbat888) [inline src/class4Strings/StringAPI.java:132]: stex menak mi hat note, result-t anyndhat poxum es.. + es anum noric es veragrum, es depqeri hamar menq 
StringBuilder enq ancel, vor append anenq + anelu u veragrelu poxaren)
bayc gract chishta, uxaki poxir string builder-ov u verj)

**Review by Smbat888 — COMMENTED** [review]
apres, es mi 2 bann el poxi u lava

File: src/class4Strings/StringAPI.java (hunk 1 above)
Comment (MagaHovhannisyan) [reply src/class4Strings/StringAPI.java:67]: Thank you for comments.
Just made all the suggested changes and merged the PR.

**Pull request merged**